deployment/bare_metal/start.sh

```diff
@@ -73,12 +74,41 @@ git pull
 export BUILD_COMMIT="$(git rev-parse HEAD)"
 
 # Generate gradido.conf from template
+# *** 1st prepare for each apiversion the federation conf for nginx from federation-template
+# *** set FEDERATION_PORT from FEDERATION_COMMUNITY_APIS and create gradido-federation.conf file
+rm -f $NGINX_CONFIG_DIR/gradido.conf.tmp
+rm -f $NGINX_CONFIG_DIR/gradido-federation.conf
+echo "===================================================================================================="
+IFS="," read -a API_ARRAY <<< $FEDERATION_COMMUNITY_APIS
+for api in "${API_ARRAY[@]}"
+do
+  export FEDERATION_APIVERSION=$api
+  # calculate port by remove '_' and add value of api to baseport
+  port=${api//_/}
+  FEDERATION_PORT=${FEDERATION_COMMUNITY_API_PORT:-5000}
+  FEDERATION_PORT=$(($FEDERATION_PORT + $port))
+  export FEDERATION_PORT
+  echo " create ngingx config: location /api/$FEDERATION_APIVERSION  to  http://127.0.0.1:$FEDERATION_PORT"
+  envsubst '$FEDERATION_APIVERSION, $FEDERATION_PORT' < $NGINX_CONFIG_DIR/gradido-federation.conf.template >> $NGINX_CONFIG_DIR/gradido-federation.conf
```

Comment:
```suggestion
  envsubst '$FEDERATION_APIVERSION, $FEDERATION_PORT' < $NGINX_CONFIG_DIR/gradido-federation.conf.template >> $NGINX_CONFIG_DIR/gradido-federation.conf.tmp
```

Please rename the generated file to something else then `.conf` - if for some reason this process is disrupted and the file stays there nginx will read it.

Reply:
renamed